include/configcat/configcatlogger.h

```diff
@@ -35,10 +37,10 @@ class ConfigCatLogger {
 
 class LogEntry {
 public:
-    LogEntry(std::shared_ptr<ConfigCatLogger> logger, LogLevel level): logger(logger), level(level) {}
+    LogEntry(std::shared_ptr<ConfigCatLogger> logger, LogLevel level, int eventId) : logger(logger), level(level), eventId(eventId) {}
     ~LogEntry() {
         if (logger && level <= logger->getLogLevel())
```

Comment:
This check seems to happen too early. `hooks->invokeOnError(message);` should be fired regardless of the logger's log level. At least this is how it's implemented in other SDKs.